pcap4j-core/src/main/java/org/pcap4j/packet/IpV6PadNOption.java

```diff
@@ -189,7 +189,6 @@ public boolean equals(Object obj) {
   @Override
   public int hashCode() {
     int result = 17;
-    result = 31 * result + dataLen;
```

Comment:
This is needed because dataLen is not always same as data.length.
Please recover this line.